traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -1224,6 +1228,9 @@ func createV3(inf *api.APIInfo, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	str := uuid.New().String()
```

Comment:
sorry, i just did a quick glance at the code. what happens on server update (PUT /servers/{id})? can the user change the XMPPID? If so, that seems bad and should be prevented if the XMPPID is driving the hashId value used in TR.

Reply:
so i guess what i'm getting at is this:

```
Create a server in traffic_portal. Observe the xmpp_id value in db. It should be set to uuid.
Edit the hostname of the created server (from step 1)
Observe the change of hostname in DB but xmpp_id remains unaffected.
```

but can i update the xmppId on a server via PUT /servers/{id}?

Reply:
Thanks for the feedback, Jeremy. I have updated the PR to handle change in XMPPID, which would be ignored and ensure hashid remains unaffected.